db/migrate/20150422201652_add_approved_to_companies.rb

```diff
@@ -0,0 +1,5 @@
+class AddApprovedToCompanies < ActiveRecord::Migration
+  def change
+    add_column :companies, :status, :string
+  end
+end
```

Comment:
:+1: Like adding in this column to the company table. Is there a way we can default this to uncontacted?